File: Flashcard.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("Flashcard", Schema = "unimind")]
public class Flashcard
{
    [Key]
    [Column("idFlashcard")]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int IdFlashcard { get; set; }

    [Column("idUsuario")]
    public int IdUsuario { get; set; }

    [Column("perguntaUsuario")]
    public string PerguntaUsuario { get; set; }
    
    [Column("respostaUsuario")]
    public string RespostaUsuario { get; set; }
}

File: app/src/main/java/com/example/unimind/data/Flashcard.kt
package com.example.unimind.data

import com.google.gson.annotations.SerializedName

data class Flashcard(
    @SerializedName("idFlashcard") val idFlashcard: Int,
    @SerializedName("idUsuario") val idUsuario: Int,
    @SerializedName("perguntaUsuario") val perguntaUsuario: String,
    @SerializedName("respostaUsuario") val respostaUsuario: String
)